<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>485</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>485</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>glBooklet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/resources/images/glBooklet.png</normaloff>:/resources/images/glBooklet.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="go_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Go!</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress_bar_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>63</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>212</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="FileSelector">
     <item row="0" column="0">
      <widget class="QLabel" name="input_file_label">
       <property name="text">
        <string>Input File:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="select_dir">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="mainwindow.qrc">
         <normaloff>:/resources/images/document-export.svg</normaloff>:/resources/images/document-export.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="scale_label">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="scale_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="scale_unit">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="selected_file"/>
     </item>
     <item row="5" column="0">
      <widget class="QSlider" name="scale_slider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>75</number>
       </property>
       <property name="sliderPosition">
        <number>75</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="selected_dir">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="output_dir_label">
       <property name="text">
        <string>Output Folder:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="select_file">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="mainwindow.qrc">
         <normaloff>:/resources/images/document-import.svg</normaloff>:/resources/images/document-import.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="test_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Print a limited number of pages, starting with the first one</string>
       </property>
       <property name="toolTipDuration">
        <number>10</number>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Print only up to page:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="test_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Print a limited number of pages, starting with the first one</string>
       </property>
       <property name="toolTipDuration">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="tittle_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>282</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Open Sans&quot;;
image: url(:/resources/images/logo32x32.png);</string>
    </property>
    <property name="text">
     <string>glBooklet</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>254</y>
      <width>210</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="pageSettings">
     <item row="0" column="1">
      <widget class="QCheckBox" name="front_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Front</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="back_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fs_left_tx_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="fs_left_tx_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-400</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="bs_left_tx_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="minimum">
        <number>-400</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="bs_left_tx_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fs_right_tx_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="fs_right_tx_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-400</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="bs_right_tx_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="minimum">
        <number>-400</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="bs_right_tx_label">
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="fs_ty_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Up&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="fs_ty_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>-575</number>
       </property>
       <property name="maximum">
        <number>585</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="bs_ty_spinbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="minimum">
        <number>-575</number>
       </property>
       <property name="maximum">
        <number>585</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="bs_ty_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Up&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_manual"/>
    <addaction name="action_donate"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/resources/images/application-exit.svg</normaloff>:/resources/images/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_manual">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/resources/images/help-about.svg</normaloff>:/resources/images/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Manual</string>
   </property>
  </action>
  <action name="action_donate">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/resources/images/help-donate.svg</normaloff>:/resources/images/help-donate.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Donate</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/resources/images/QtIcon.png</normaloff>:/resources/images/QtIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scale_spinbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scale_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scale_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
